apps/odo/mix.exs

```diff
@@ -0,0 +1,62 @@
+defmodule Odo.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :odo,
+      version: "0.2.0",
```

Comment:
might want to check and make sure we don't already have a release tagged with `0.2.0` and if we do, rev this slightly.

Reply:
Bumped to 0.3.0